osquery/tables/system/windows/kva_speculative_info.cpp

```diff
@@ -1,10 +1,11 @@
 /**
- *  Copyright (c) 2014-present, Facebook, Inc.
- *  Copyright 2018 Alex Ionescu.
- *  All rights reserved.
+ * Copyright (c) 2014-present, The osquery authors
+ * Copyright 2018 Alex Ionescu.
```

Comment:
Ditto

Reply:
This one has always been there. So I preserved it.

https://github.com/osquery/osquery/pull/4027/files#diff-7797297b84012df442212204019bd468R3